tests/blockchain/test_blockchain.py

```diff
@@ -1816,7 +1816,7 @@ async def test_conditions(self, empty_blockchain, opcode, with_garbage, bt):
         wt: WalletTool = bt.get_pool_wallet_tool()
 
         tx1: SpendBundle = wt.generate_signed_transaction(
-            10, wt.get_new_puzzlehash(), list(blocks[-1].get_included_reward_coins())[0]
+            10, wt.get_new_puzzlehash(), blocks[-1].get_included_reward_coins()[0]
```

Comment:
This is less obviously a good answer, but I'll mention a couple ideas here and maybe we have a conversation.

This can get 'some' element from any iterable.

```suggestion
            10, wt.get_new_puzzlehash(), next(iter(blocks[-1].get_included_reward_coins()))
```

Reply:
As in existing code before this PR depends on order in sets?  That can't be expected to be consistent.  Python hashing is seeded differently each run to avoid some attacks.  But maybe I'm misunderstanding.  I think that iterating a set twice for example would yield the same order each time.

Reply:
https://github.com/Chia-Network/chia-blockchain/pull/16855/files#r1398231751